src/main/java/org/opensearch/security/privileges/PrivilegesEvaluator.java

```diff
@@ -425,6 +427,13 @@ public PrivilegesEvaluatorResponse evaluate(final User user, String action0, fin
                         presponse.allowed = true;
                         return presponse;
                     }
+
+                    if(request instanceof ResolveIndexAction.Request) {
```

Comment:
can this be unified to `Replaceable`? There is lots of duplicate code.

Reply:
Can only apply to the one for resolve API

The method `indicesOptions` has different signature defined with `Replaceable`

```java
public interface IndicesRequest {
    String[] indices();

    IndicesOptions indicesOptions();

    default boolean includeDataStreams() {
        return false;
    }

    public interface Replaceable extends IndicesRequest {
        IndicesRequest indices(String... var1);
    }
}
```

Reply:
No need to provide definition of `Replaceable`